lms/urls.py

```diff
@@ -267,6 +267,11 @@
             'courseware.module_render.xblock_resource',
             name='xblock_resource_url'),
 
+        # xblock Rendering
+        url(r'^xblock/{usage_key_string}$'.format(usage_key_string=settings.USAGE_KEY_PATTERN),
```

Comment:
Thanks @nasthagiri. I thought that this was the API that returned the HTML along with CSS and JavaScript in a JSON payload. I see now that it is a view that returns HTML, so I am indeed happier with having this not be an API. Sorry for the confusion.